moveit_ros/visualization/rviz_plugin_render_tools/src/render_shapes.cpp

```diff
@@ -156,11 +156,14 @@ void RenderShapes::renderShape(Ogre::SceneNode* node, const shapes::Shape* s, co
 
     case shapes::OCTREE:
     {
-      OcTreeRenderPtr octree(new OcTreeRender(static_cast<const shapes::OcTree*>(s)->octree, octree_voxel_rendering,
-                                              octree_color_mode, 0u, node));
-      octree->setPosition(position);
-      octree->setOrientation(orientation);
-      octree_voxel_grids_.push_back(octree);
+      if (octree_voxel_rendering != OCTOMAP_DISABLED)
```

Comment:
So.. when an already rendered OcTree is getting disabled, this won't take care of removing it, right?

Reply:
I had thought so too, but switching to "Disabled" in RViz does in fact hide a previously rendered OcTree

Reply:
nice!